assets/component-volume-pricing.css

```diff
@@ -49,10 +49,21 @@ volume-pricing show-more-button {
 }
 
```

Comment:
I think theres a regression here: https://screenshot.click/19-57-wlszg-nl9px.png

Add something through quick order list, add from the PDP then use the trash icon in the quick order list and this happens

Reply:
Let me sum up here, since we still don't see a pattern and we are not sure if this bug was introduced here we will address it as a follow up.